adoc/headers/accessorBuffer.h

```diff
@@ -44,7 +44,7 @@ class accessor {
   using const_reverse_iterator = std::reverse_iterator<const_iterator>;
   using difference_type =
       typename std::iterator_traits<iterator>::difference_type;
-  using size_type = size_t;
+  using std::size_type = std::size_t;
```

Comment:
```suggestion
  using size_type = std::size_t;
```

Check for other cases where `std::` was added before `size_type`.  I assumed this happened because you had a search/replace for "size_t" and that matched "size_type".